.github/workflows/build-voyager.yml

```diff
@@ -10,8 +10,11 @@ on:
     inputs:
 
 jobs:
+  prepare-versions:
+    uses: ./.github/workflows/prepare-versions.yml
```

Comment:
Is there a way to only call this workflow once, this is being called before every workflow, so a lot of duplicate prepare-versions are running